tests/cypress/integration/canvas3d_functionality/case_86_canvas3d_functionality_move_image_button.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const_canvas3d';
+
+context('Canvas 3D functionality. "Move the image" button interaction.', () => {
+    const caseId = '86';
+    const screenshotsPath = 'cypress/screenshots/canvas3d_functionality/case_86_canvas3d_functionality_move_image_button.js';
+    const cuboidCreationParams = {
+        labelName: labelName,
+    };
+
+    before(() => {
+        cy.openTask(taskName)
+        cy.openJob();
+        cy.wait(1000); // Waiting for the point cloud to display
+        cy.create3DCuboid(cuboidCreationParams);
+        cy.get('.cvat-canvas3d-perspective').screenshot('canvas3d_perspective_after_add_cuboid');
+        ['topview', 'sideview', 'frontview'].forEach((view) => {
+            cy.get(`.cvat-canvas3d-${view}`)
+                .find('.cvat-canvas3d-fullsize')
+                .screenshot(`canvas3d_${view}_after_add_cuboid`);
+        });
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Click to "Move the image" button. The cuboid on the top/side/front view should be hidden.', () => {
+            cy.get('.cvat-move-control').click();
```

Comment:
Would be useful to check that mousemoving on the canvas (over a cuboid) does not lead to activating an object. When usual mode (cursor) leads to it.

Reply:
Added the necessary steps and checks.